### DISCUSSION TITLE
How to switch from Play CDN to production build?

### DISCUSSION THREAD

**Discussion by coolymoon** [tailwindlabs/tailwindcss#7622] [Help]
I am using Play CDN for local development, and would like to use PostCSS to build a production version

Basically every time I need to manually replace the following line in `index.html`
`<script src="https://cdn.tailwindcss.com"></script>`
with 
`<link href="/dist/main.css" rel="stylesheet">`
before pushing to production site

What is the best way to do that automatically?

**Comment by supto51** [comment]
You need to use build tools like  webpack, Rollup, Vite, and Parcel. You can use vite because it's need zero configuration.
